integration/ui/satellite/tardigrade_branding_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"testing"
+	"time"
+
+	"github.com/go-rod/rod"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/integration/ui/uitest"
+	"storj.io/storj/private/testplanet"
+)
+
+func TestForTardigradeBranding(t *testing.T) {
+	uitest.Run(t, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, browser *rod.Browser) {
+		loginPageURL := planet.Satellites[0].ConsoleURL() + "/login"
```

Comment:
hopefully we will soon move the /login page completely out of the tardigrade branding repo. Eventually we want to move everything out of there.

Login page should already be removed from the branding repo for all user-facing satellites (us1, ap1, eu1). Registration too, I think. Eventually we want to move all code out of the branding repo, but I don't have a timeline yet. These kinds of tests would still be useful for the pages that are remaining.